css-brochure-form/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+  <h1> LEARN MORE </h1>
+  <form action="http://www.example.com/signup.php">
+    <p>COMPANY EMAIL</p>
+    <input type="text" name="email" size="100" maxlength="30" placeholder="nhu@learningfuze.co"/>
+    <p>NAME</p>
+    <input type="text" name="name" size="100" maxlength="30" placeholder="nhu pham" />
+    <p>WHERE ARE YOU HIRING?</p>
+    <input type="text" name="where" size="100" maxlength="30" placeholder="City, Country" />
+    <p>HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</p>
+    <div>
+      <select name="amount">
+        <option value="1-2">1-2 Engineers</option>
+        <option value="2-4">2-4 Engineers</option>
+        <option value="4-6">4-6 Enginerrs</option>
+      </select>
+    </div>
+    <div class="button">
+      <button type="button">SUBMIT</button>
```

Comment:
if you use the `button` type for a button, it will not work in a form to submit the form. the default type of a button in a form is `submit` so either don't use a type for a button in a form, or set it explicitly to type `submit`